src/librustc_middle/hir/map/mod.rs

```diff
@@ -577,17 +587,7 @@ impl<'hir> Map<'hir> {
     /// Whether the expression pointed at by `hir_id` belongs to a `const` evaluation context.
     /// Used exclusively for diagnostics, to avoid suggestion function calls.
     pub fn is_const_context(&self, hir_id: HirId) -> bool {
```

Comment:
Just a personal opinion, but what about renaming this to `is_expr_inside_const_context`? I feel like it's not clear when to use this vs. `body_const_context().is_some()` at the moment.

Reply:
That a good opinion, changed it to `is_inside_const_context`.